yarn.lock

```diff
@@ -3038,6 +3038,13 @@
   dependencies:
     "@types/react" "*"
 
+"@types/react-is@^18.2.0":
```

Comment:
A separate dependency version has to be introduced only because we are not maintaining the `benchmark` package and it has `"@material-ui/core": "^5.0.0-beta.5",` dependency. 😢 
@m4theushw Do we have any plans for this package?
- Would it be a big job to at least refresh its dependencies?
- Would it make sense to pull it out from the ashes and make it active?